build.assets/tooling/cmd/build-apt-repos/apt_repo_tool.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+import (
+	"io/fs"
+	"log"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+)
+
+type AptRepoTool struct {
+	config       *Config
+	aptly        *Aptly
+	s3Manager    *s3manager
+	supportedOSs *map[string][]string
```

Comment:
This doesn't need to be a pointer (maps are already pointers).

Reply:
Thanks - I was unaware of that. Updated